Packs/HPE_Service_Manager/Integrations/ServiceManager/Service_Manager.yml

```diff
@@ -1248,4 +1368,24 @@ script:
       description: Device status.
     - contextPath: HPSM.Devices.UpdatedBy
       description: The last user who updated the device.
-fromversion: 5.0.0
+    description: Find and return a device by ID.
+  - name: hpsm-create-request
+    arguments:
+    - name: callbackcontactname
+    - name: contactname
+      required: true
+    - name: title
+      required: true
+    - name: purpose
+    outputs:
```

Comment:
Check that all relevant fields are shown here (`Purpose` and `CallbackType` aren't, they should be?).